lib/src/commonMain/kotlin/org/reduxkotlin/Definitions.kt

```diff
@@ -52,3 +58,26 @@ fun <State> middleware(dispatch: (Store<State>, next: Dispatcher, action: Any) -
             }
         }
     }
+
+
+/**
+ * Convenience function for creating a [ReducerForActionType]
+ * usage:
+ *   sealed class LoginScreenAction
+ *   data class LoginComplete(val user: User): LoginScreenAction()
+ *
+ *   val loginReducer = sealedReducer<AppState, LoginAction> { state, action ->
+ *       when(action) {
```

Comment:
Might be worth showing how 2 sealedReducers can be combined and passed to a Store

Reply:
added